libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/utils/TrainUtils.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.ai.qna.utils;
+
+import java.util.concurrent.CompletableFuture;
+
+import com.microsoft.bot.ai.qna.models.FeedbackRecords;
```

Comment:
These imports should be go at the beginning

Reply:
Done